src/components/notifications/index.js

```diff
@@ -116,6 +117,7 @@ const NotificationFormList = {
     "ServerChan": ServerChan,
     "ZohoCliq": ZohoCliq,
     "gtxmessaging": GtxMessaging,
+    "Cellsynt": Cellsynt
```

Comment:
nit:
https://github.com/louislam/uptime-kuma/pull/4558#discussion_r1538361274

```suggestion
    "Cellsynt": Cellsynt,
```